packages/psss/src/containers/Tables/AlertsTable/getColumns.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  AlertMenuCell,
+  CountryLinkCell,
+  StartDateCell,
+  SyndromeCell,
+  WeekAndDateCell,
+} from '../../../components';
+
+const createCountryColumn = alertCategory => {
+  const CellComponent = ({ organisationUnit }) => (
+    <CountryLinkCell
+      target={`/alerts/${alertCategory}/${organisationUnit}`}
```

Comment:
As far as I remember there wasn't a requirement to do this. I actually think it's a nice feature but does make some other things like the date and the navigation to open the panel confusing. Might be worth touching base with Dave.